beetsplug/check.py

```diff
@@ -70,6 +70,7 @@ def __init__(self):
             "import": True,
             "write-check": True,
             "write-update": True,
+            "auto-fix": False,  # Will automatically fix integrity errors on import
```

Comment:
```suggestion
            "auto-fix": False,
```

Let’s be consistent and not document the purpose of the setting here. It’s already sufficiently documented in the readme